### DISCUSSION TITLE
[FR] Support the Twig {% deprecated %} tag

### DISCUSSION THREAD

**Discussion by khalwat** [craftcms/cms#10181] [Ideas] [answered]
Since Twig 2.6, Twig has supported the [`{% deprecated %}` tag](https://twig.symfony.com/doc/3.x/tags/deprecated.html). Currently, it does nothing in Craft (despite it being [documented as doing something](https://craftcms.com/docs/3.x/dev/tags.html)). Looks like a small amount of work needs to be done to support it:

https://twig.symfony.com/doc/3.x/recipes.html#deprecation-notices

...but it would be nice if it was supported, and just passed along the deprecation notices to `Craft::$app->getDeprecator()->log()`

PR created: https://github.com/craftcms/cms/pull/10182

**Comment by brandonkelly** [comment, marked as answer]
Added for the next release! Ended up just replacing Twig’s built-in `{% deprecated %}` tag with a custom one.

**Reply by brandonkelly** [reply]
Craft 3.7.24 is out now with that change ✨

**Comment by khalwat** [comment]
haha or that :)
